app/views/figures/show.html.erb

```diff
@@ -5,12 +5,14 @@
 <p>Nationality: <%= @figure.nationality %></p>
 <p>Claim to fame: <%= @figure.claim_to_fame %></p>
 
-<p>Average Rating: <%= @average_rating %></p>
-
-<h3>Submit a Rating</h3>
+<% if @average_rating.is_a?(Numeric)%>
```

Comment:
why checking to see if it's a number instead of seeing if it's empty? mostly curious